dist/Dockerfile.build/Dockerfile

```diff
@@ -8,18 +8,15 @@ RUN yum -y groupinstall 'Development Tools'
 RUN yum -y install openssl-devel
 
 ENV HOME="/root"
-
-# build: Rust stable toolchain
-RUN \
-  mkdir $HOME/rust && \
-  curl https://static.rust-lang.org/dist/rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz | \
-  tar -xzvf - -C $HOME/rust --strip 1 && \
-  $HOME/rust/install.sh; \
-  rm -rf $HOME/rust
-
-# add cargo binaries to PATH
 ENV PATH="${HOME}/.cargo/bin:${PATH}"
 
+# build: Rust stable toolchain + 2 previous versions
+RUN mkdir $HOME/rust && \
```

Comment:
This `mkdir` is leftover and not required for rustup. This made me wonder if `rustup` cleans up after itself, or if the installation files end up in the image.

Reply:
Good catch, fixed

Reply:
Thanks. I checked the rustup script and it does clean up after itself so we're good.